quipucords/fingerprinter/__init__.py

```diff
@@ -84,19 +81,18 @@ def process_fact_collection(sender, instance, **kwargs):
     instance.save()
 
 
-def _process_sources(raw_facts):
+def _process_sources(fact_collection):
     """Process facts and convert to fingerprints.
 
-    :param raw_facts: Collected raw facts for all sources
+    :param json_fact_collection: Collected raw facts for all sources
```

Comment:
param name in description doesn't match param

Reply:
Good catch.  Fixed.